web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -16,77 +20,91 @@ import Divider from "@mui/material/Divider";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
 import SkeletonLoader from "../components/skeleton_loader/SkeletonLoader";
 
-const PREFIX = 'ReceivedRequestsPage';
+const PREFIX = "ReceivedRequestsPage";
 const classes = {
   pageTitle: `${PREFIX}-pageTitle`,
   textField: `${PREFIX}-textField`,
   searchField: `${PREFIX}-searchField`,
   formControl: `${PREFIX}-formControl`,
   notFoundMessage: `${PREFIX}-notFoundMessage`,
   expandClose: `${PREFIX}-expandClose`,
-  expandOpen: `${PREFIX}-expandOpen`
+  expandOpen: `${PREFIX}-expandOpen`,
 };
 
-const Root = styled('div')({
+const Root = styled("div")({
   [`& .${classes.pageTitle}`]: {
     paddingRight: "0.4em",
     minWidth: "330px",
-    ['@media screen and (max-width: 600px)']: { // eslint-disable-line no-useless-computed-key
+    ["@media screen and (max-width: 600px)"]: {
+      // eslint-disable-line no-useless-computed-key
```

Comment:
This is why your build is failing...has to be on the same line. @jahanner

Reply:
Gratzie